app/SideBar.js

```diff
@@ -37,6 +44,13 @@ const SideBarIcon = ({icon,version}) =>{
         text = 'Notes 📝';
     }else if(version === 'robot'){
         text = 'Ai Generator 🦾';
+    }else if(version === 'user'){
```

Comment:
try and avoid using plain strings and convert these into enums or constant objects in a separate folder like
constants/enums/MyHotEnum.ts